src/components/Dropdown/Dropdown.scss

```diff
@@ -72,16 +72,6 @@ $border-width: 1px;
         outline: none;
         box-shadow: inset 0 0 0 2px var(--color-accent);
     }
-
-    &:active {
-        color: var(--color-light);
-
-        background-color: var(--color-accent);
-    }
-
-    &.moonstone-opened {
-        @extend %dropdown-opened;
-    }
```

Comment:
It seems we need to keep this class to keep the blue border while the dropdown menu is opened. Currently when we use a multiple dropdown and I select an item the border disappears.

Reply:
dropdown-opened was only about putting a blue background : 
```
%dropdown-opened {
    color: var(--color-light);

    background-color: var(--color-accent);
}
```

The border is coming from somewhere else ?

Reply:
Currently, we only display the border for the `outlined` variant and not for the `ghost` variant; and I think we should have the same style for both variants.

As we replaced the blue background with a blue border when the dropdown menu is opened, I suggest using the class `.dropdown-opened`  to manage the style when the menu is opened.